### ISSUE TITLE
Feature Request: Include types

### ISSUE THREAD

**Issue by cinderblock** [fivdi/pigpio#103]
I'm wondering if you would accept a PR that brings the types from DefinitelyTyped into this repo. It is a pretty small penalty on published/repository size but would make it significantly easier for people to use and keep the two APIs in sync.

For instance, the `configureInterfaces` function is not available in the DefinitelyTyped package. I'm creating a PR there to add the missing type but it would have been nice if it were included in #82.

Basically we would just need to copy the types from the DefinatelyTyped package and add a `types` field to the published `package.json`. I'd be happy to take up the charge on this. Thoughts?

**Comment by fivdi** [comment]
> I'm wondering if you would accept a PR that brings the types from DefinitelyTyped into this repo.

Yes, such a PR would be accepted.
